src/Viewer/components/MenuBar/MenuBar.tsx

```diff
@@ -59,6 +86,14 @@ const MenuBar = ({
     const handleShowHelp = () => {
         setShowHelp(true);
     };
+    const handleTimezoneChange = (event, newTimezone) => {
+        if (null === newTimezone) return;
```

Comment:
We don't call `handleTimezoneChange()` with `newTimezone === null`, right?

Reply:
handleTimezoneChange is also called when a button is released. It will generate a null signal, which needs to be caught.

Reply:
In ToggleButtonGroupProps.d.ts: `onChange?: (event: React.MouseEvent<HTMLElement>, value: TValue extends Array<infer U> ? Array<U> : TValue | null) => void;`

Reply:
Gotcha. I ignored the fact that in exclusive mode the toggle button doesn't enforce a minimum number of selections. The null check and the handling in this callback function is reasonable then.